src/renderer/components/Modal/ModalContent.js

```diff
@@ -63,8 +63,6 @@ const ModalContent: React$ComponentType<Props> = React.forwardRef(function Modal
     };
   }, [containerRef, onHeightUpdate]);
 
-  useEffect(() => {}, [isScrollable]); // WAT ?
```

Comment:
:wat:

Reply:
I removed this it was standing there doing nothing :man_shrugging: